modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java

```diff
@@ -751,45 +726,16 @@ private DLFileEntry _getDLFileEntry(
 			String externalReferenceCode, Long siteId)
 		throws Exception {
```

Comment:
Does it make sense to have this method that just makes a request to another service and do nothing else?